src/tests/stress/pedsim/parallel_big.enc

```diff
@@ -0,0 +1,30 @@
+import Ped_util.Global_funs
```

Comment:
please do not include binary files, i.e. remove the `src/tests/stress/pedsim/parallel_big` file